tests/test_reports/test_trajectories_report.py

```
@@ -0,0 +1,387 @@
+    )
+    _write_jsonl(bench / "model_traffic.jsonl", [_wire(0, 0, prompt=10, completion=5)])
+    report = json.loads(generate_trajectories_report(tmp_path, enrich=True).read_text())["benchmarks"][0]
+    enr = report["enrichment"]
+    assert enr["trials_spliced_1_to_1"] == 0
+    assert enr["trials_with_unmatched_calls_stashed"] == 1
+    # No step-level backfill (partial splice would misattribute)
+    assert enr["steps_backfilled"]["prompt_tokens"] == 0
+    # The wire call landed in extra.captured_model_calls
+    enriched = json.loads((bench / "trajectories_enriched.jsonl").read_text().splitlines()[0])
+    stash = enriched["trajectory"][0].get("extra", {}).get("captured_model_calls")
+    assert isinstance(stash, list) and len(stash) == 1
```

Ruff (0.15.17)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)